realm/realm-library/src/androidTest/java/io/realm/RealmListTests.java

```diff
@@ -783,8 +783,8 @@ public void realmMethods_onDeletedLinkView() {
                     case MAX_DATE: results.maxDate(CyclicType.FIELD_DATE); break;
                     case DELETE_ALL_FROM_REALM: results.deleteAllFromRealm(); break;
                     case IS_VALID:              // Does not throw.
-                    case IS_MANAGED: continue;  // Does not throw.
-                    case IS_FROZEN: break;      // Does not throw
+                    case IS_MANAGED:            // Does not throw.
+                    case IS_FROZEN: continue;   // Does not throw
```

Comment:
Yes, but it is because it was wrong and it was my mistake. I inadvertently introduced this as a bug in the first task.

Background info: this is due to the addition of the `Freezable` interface, which has changed the operations we can perform on collections - now `freeze` doesn't exist as an `OrderedRealmCollection` operation.

I was under the impression that it was OK to implement `Freezable` from the bottom (i.e. in `RealmList`, `RealmResults`) and not from the top (i.e. `OrderedRealmCollection`). Maybe this assumption is wrong?

Reply:
I would say freezable belong to the top-level classes, so the tests should probably also be there. I.e. it belongs to the `CollectionTests` class as it should be the `RealmCollection` that implements Freezable.

Reply:
Sure. I will change it in the next PR for better visibility.